util/legacy.ruskInputsToInputs fails with segmentation violation

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xdf439e]

goroutine 29 [running]:
github.com/dusk-network/dusk-blockchain/pkg/util/legacy.ruskInputsToInputs(0xc00060e208, 0x1, 0x1, 0xc0001db940, 0x20, 0x20, 0x1, 0x20)
	/feature-588/dusk-blockchain/pkg/util/legacy/conversions.go:404 +0x1fe
github.com/dusk-network/dusk-blockchain/pkg/util/legacy.RuskTxToStandard(0xc0007c94d0, 0x7f48d2483460, 0x0, 0xc000813560)
	/feature-588/dusk-blockchain/pkg/util/legacy/conversions.go:234 +0x1e2
github.com/dusk-network/dusk-blockchain/pkg/util/legacy.RuskBidToBid(0xc0007cc6e0, 0x3, 0x3, 0xc000813560)
	/feature-588/dusk-blockchain/pkg/util/legacy/conversions.go:302 +0x61
github.com/dusk-network/dusk-blockchain/pkg/util/legacy.ContractCallsToTxs(0xc000860e00, 0x3, 0x4, 0x0, 0x0, 0xc0007d78e8, 0xc3c5fb, 0x1a460e8)
	/feature-588/dusk-blockchain/pkg/util/legacy/conversions.go:151 +0x13a
github.com/dusk-network/dusk-blockchain/pkg/util/ruskmock.(*Server).ExecuteStateTransition(0xc0001b82a0, 0x1366ca0, 0xc000813320, 0xc000888c80, 0xc0001b82a0, 0xc0001bfa40, 0xc000188700)
	/feature-588/dusk-blockchain/pkg/util/ruskmock/server.go:150 +0x81
github.com/dusk-network/dusk-protobuf/autogen/go/rusk._Rusk_ExecuteStateTransition_Handler(0x11b20a0, 0xc0001b82a0, 0x1366ca0, 0xc000813320, 0xc0001bfa40, 0x0, 0x1366ca0, 0xc000813320, 0xc00089e000, 0xedf)
	/home/tech/go/pkg/mod/github.com/dusk-network/dusk-protobuf@v0.2.8-0.20200528094759-4457b6e5f466/autogen/go/rusk/rusk.pb.go:2332 +0x2fd
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001ba000, 0x13704e0, 0xc0001bc900, 0xc00083ea00, 0xc000186a50, 0x1a1adf0, 0x0, 0x0, 0x0)
	/home/tech/go/pkg/mod/google.golang.org/grpc@v1.29.0/server.go:1082 +0x955
google.golang.org/grpc.(*Server).handleStream(0xc0001ba000, 0x13704e0, 0xc0001bc900, 0xc00083ea00, 0x0)
	/home/tech/go/pkg/mod/google.golang.org/grpc@v1.29.0/server.go:1405 +0x12be
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc0005fc200, 0xc0001ba000, 0x13704e0, 0xc0001bc900, 0xc00083ea00)
	/home/tech/go/pkg/mod/google.golang.org/grpc@v1.29.0/server.go:746 +0xc9
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/home/tech/go/pkg/mod/google.golang.org/grpc@v1.29.0/server.go:744 +0xb9
```

--- a/pkg/util/legacy/conversions.go
+++ b/pkg/util/legacy/conversions.go
@@ -396,6 +396,8 @@ func ruskInputsToInputs(inputs []*rusk.TransactionInput) (transactions.Inputs, e
 	sInputs := make(transactions.Inputs, len(inputs))
 
 	for i, input := range inputs {
+		sInputs[i] = new(transactions.Input)
+
 		buf := bytes.NewBuffer(input.Nullifier.H.Data)
 		keyImageBytes := make([]byte, 32)
 		if err := encoding.Read256(buf, keyImageBytes); err != nil {

